model/pom.xml

```diff
@@ -215,6 +204,7 @@
             </database>
             <target>
               <packageName>keywhiz.jooq</packageName>
+              <directory>src-generated/main/java</directory>
             </target>
```

Comment:
By default, Jooq jumps generated code under the target/ folder, which is the standard location for Maven's build outputs. However, since it's generally considered a best practice to never check in anything in that folder, we're redirecting Jooq's output to our new src-generated folder.